src/helpers/wrapRootElement.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from '@oceanprotocol/lib/dist/node/utils/ConfigHelper'
 import { UserPreferencesProvider } from '../providers/UserPreferences'
 import PricesProvider from '../providers/Prices'
+import ApolloClentProvider from '../providers/ApolloClientProvider'
```

Comment:
small typo: `ApolloClentProvider` should be `ApolloClientProvider`

Reply:
also, now the app has 2 `ApolloProvider`, one added in `App.tsx` and one here. So should be removed again from `App.tsx` if all works when having the provider here